src/Umbraco.Cloud.StorageProviders.AzureBlob/AzureBlobOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Umbraco.Cloud.StorageProviders.AzureBlob
+{
+    /// <summary>
+    /// The Azure Blob options used on Umbraco Cloud.
+    /// </summary>
+    internal class AzureBlobOptions
+    {
+        /// <summary>
+        /// Gets or sets the endpoint.
+        /// </summary>
+        /// <value>
+        /// The endpoint.
+        /// </value>
+        [Required]
+        public string Endpoint { get; set; } = null!;
```

Comment:
It would make more sense to change this property (and related environment variable) name to:
```suggestion
        public string BlobEndpoint { get; set; } = null!;
```

Reply:
The property is already nested in `AzureBlob` options, and the env. variable is also named `Umbraco:Cloud:Storage:AzureBlob:Endpoint` so it's already given it's the blob endpoint and I don't see any reason to change it

Reply:
It might indeed seem a bit repetitive, but my reasoning was to align it with the connection string key `BlobEndpoint=`.

Reply:
I see your point, but we are already provisioning with the endpoint variable and don' want to change that now, so we'll keep it as it is